assets/src/application/ui/display/EntityIDs/index.tsx

```diff
@@ -17,15 +16,14 @@ export interface EntityIdProps {
 }
 
 const EntityIDs: React.FC<EntityIdProps> = React.memo(({ dbid, guid, align = 'left' }) => {
-	const htmlClass = classNames('ee-entity-ids', 'ee-focus-priority-9', {
+	const className = classNames('ee-entity-ids', 'ee-focus-priority-9', {
 		'ee-align-lft': align === EntityIdAlignment.LEFT,
 		'ee-align-rgt': align === EntityIdAlignment.RIGHT,
 	});
+
 	return (
-		<div className={htmlClass}>
+		<div className={className}>
 			<span className={'ee-entity-dbid'}>{dbid}</span>
-			<span className={'ee-entity-id-separator'}>{':'}</span>
-			<span className={'ee-entity-guid'}>{shortenGuid(guid)}</span>
```

Comment:
experimented with this approach as well:
![image](https://user-images.githubusercontent.com/1477453/86626299-2c43d580-bfcf-11ea-94ae-078f28b12753.png)

Reply:
I had done that too at some point, but I've also had some other feature ideas that may require an extremely prominent spot in the UI like that. For example, I have a branch that adds a `locked` boolean property to the dates and tickets, and displays a lock icon in that same top corner of the sidebar where you put the ID. when toggled on, the lock  prevents those entities from being edited via automated processes like REM. I think it's a good chance that we will end up adding that but I didn't feel it was critical for REM + EDTR and so I left it out for now.